src/components/Register/Register.js

```diff
@@ -100,60 +142,80 @@ function RegisterContainer({ data, navigation }) {
         return { ...all, input };
       },
     );
-    try {
-      const request = {
-        callbacks,
-        authId: data.authId,
-        authServiceId: data.authServiceId,
-        header: data.pageHeader,
-        description: data.pageDescription,
-      };
+    const request = {
+      callbacks,
+      authId: data.authId,
+      authServiceId: data.authServiceId,
+      header: data.pageHeader,
+      description: data.pageDescription,
+    };
+    const response = await ForgeRockModule.next(JSON.stringify(request));
 
-      const response = await ForgeRockModule.next(JSON.stringify(request));
-      if (response.type === 'LoginSuccess') {
-        setAuthentication(true);
-        navigation.navigate('Todos');
-      } else if (response.type === 'LoginFailure') {
-        // handle failure
-      }
-    } catch (err) {
+    if (response.type === 'LoginSuccess') {
+      setData(response);
+      setAuthentication(true);
+      setLoading(false);
+      navigation.navigate('Home');
+    } else {
+      setData(response);
+      setAuthentication(false);
       setLoading(false);
     }
   };
-
-  return (
+  console.log(data);
+  return loading ? (
+    <Loading message={'Checking your session'} />
+  ) : (
     <Box safeArea flex={1} p={2} w="90%" mx="auto">
-      <Text>Register</Text>
+      <Center marginBottom={4}>
+        <Icon name="account-plus" size={72} color={'#c0c9d5'} />
```

Comment:
Let's try to avoid using color values directly and use references to theme colors like primary, secondary ... this helps aid theming without having to directly edit values in the component files.